main.cc

```diff
@@ -2183,6 +2223,8 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
 
             supervisor::notify("serving");
 
+            ::sigquit_handler sigquit_handler(db);
+
```

Comment:
Might as well launch it earlier. Consider a problem in reading the system tables (which I once had - during boot we flush some memtable, then we read, too many sstables, reads fail, crash, loop with one more sstable)

Reply:
ok, I'll move it right to where we init the database (cannot move it up earlier).

Reply:
Initialization is moved to right after database is initialized in v2.